src/components/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
I know the errors this ignores are extremely tedious, but we should be careful about ignoring them. We might have to ask the client exactly how strict we need to be with the accessibility rules and stuff